### ISSUE TITLE
NTR: Renal biopsy: proximal convoluted tubules abnormal (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9341]
New term request Renal biopsy: proximal convoluted tubules abnormal (elthat_130811194614): Absent; decreased; primitive

**Comment by pnrobinson** [comment]
@MickeySegal
Hi @avizrosenberg @yongqunh can we touch bases on this term?

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000254
- label: Abnormal proximal convoluted tubule morphology
- parents: [Abnormal renal tubule morphology (HP:0000091)](https://hpo.jax.org/app/browse/term/HP:0000091)
- pmid: 
- def: Any structural anomaly of the proximal convoluted tubule, which is the first tubular component of the nephron and arises as a continuation of the Bowman space from the tubular pole of the renal corpuscle. The proximal convoluted tubule is located within the renal cortex.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9341
- synonyms:

**Comment by yongqunh** [comment]
This term and its definition look good to me. 
The proximal tubule is a segment of the nephron in kidneys: https://en.wikipedia.org/wiki/Proximal_tubule .  
The proximal convoluted tubule (PCT)  does have the disorder:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4953774/
